src/utils/common.js

```diff
@@ -91,70 +98,70 @@ export function formatMoney(
   }
 
   return (c ? num.replace('.', c) : num).replace(new RegExp(re, 'g'), `$&${s || ','}`);
-}
+};
 
-export function parseNumber(amount: string = '0') {
-  let strg = amount.toString() || '';
+export const parseNumber = (amount: string) => {
```

Comment:
Are you sure we're passing string here as a param? I think in some places BN instance could occure

Reply:
The `= '0'`, should not have any effect on a BN being passed, or does it? :thinking:

Reply:
`= '0'` saves from undefined values, the next line `let strg = amount.toString() || '';` was saving from BigNumbers passed in

Reply:
Updated and added a couple of tests covering those scenarios to make sure we have the expected behaviour.